web/sites/default/config/webform.webform.contact_elected.yml

```diff
@@ -292,68 +304,100 @@ elements: |-
         - xor
         - ':input[name="purpose"]':
             value: provide_feedback_specific_elected
+        - xor
+        - ':input[name="purpose"]':
+            value: provide_feedback_council
     public_records_statement_text:
       '#type': webform_markup
       '#markup': 'Information you provide to the City is a public record and may be subject to release under Oregon&rsquo;s <a data-renderer-mark="true" href="https://www.doj.state.or.us/oregon-department-of-justice/public-records/attorney-generals-public-records-and-meetings-manual/i-public-records/#:~:text=Under%20Oregon''s%20Public%20Records%20Law,committee%20of%20the%20Legislative%20Assembly" title="https://www.doj.state.or.us/oregon-department-of-justice/public-records/attorney-generals-public-records-and-meetings-manual/i-public-records/#:~:text=Under%20Oregon''s%20Public%20Records%20Law,committee%20of%20the%20Legislative%20Assembly">Public Records Law</a>. This law classifies certain information as available to the public on request. See our <a data-renderer-mark="true" href="/help/about/privacy" title="https://www.portland.gov/help/about/privacy">privacy statement</a> for more information.'
   hidden_container:
     '#type': container
-    '#attributes':
-      class:
-        - visually-hidden
+    voting_electeds_entity:
+      '#type': webform_entity_checkboxes
+      '#title': 'Voting electeds'
+      '#default_value':
+        - '71'
```

Comment:
@jookyg , is there a way to make this value pull from a view? That would allow us to use the "active" and "voting" electeds group values rather than hard-coded group IDs. 🤔

Reply:
Without doing some custom development, I wasn't able to figure out a way to pull in a view and have it render as a comma delimited list. I could create a view that returned the email addresses of all current voting electeds, but I couldn't find a way to render it as a list without row containers. The only way I could find was to render the voting electeds view as a checkbox list, and the boxes have to be checked in order to retrieve the underlying entity and email address field value. There may still be a way to do it, but given that this is a minor change that only has to happen every 2 years, it doesn't seem worthwhile. Except that we need to be able to remember to do it when new electeds come in.

Reply:
Yes, I was able to use a computed twig field to create a computed confirmation message and display that on the conf page.